workshop 01 - Introduction to Quantum Computing/intro-to-qsharp.ipynb

```diff
@@ -5,6 +5,8 @@
    "metadata": {},
```

Comment:
This actually belongs to some other commit which you accidentally wrapped in. Please, remove it through rebasing.